canbench-bin/src/fmt.rs

```diff
@@ -27,10 +30,19 @@ pub(crate) fn fmt_human_i64(value: i64) -> String {
     }
 }
 
-pub(crate) fn fmt_percent(value: f64) -> String {
+/// Format a floating-point number with a sign and two decimal places.
+/// This is used for displaying percentage changes in human-readable format
+/// with a sign in the short table report.
+pub(crate) fn fmt_human_percent(value: f64) -> String {
     if value.abs() < 0.01 {
-        format!("{:.2}%", value)
+        format!("{:.2}%", value) // Don't show sign for small values.
```

Comment:
Why this exception?

Reply:
my personal preference, I find it strange to see zero value with a plus sign `+0.00%`, I'd much prefer `-0.01%`, `0.00%` and `+0.01%`.

Reply:
I see: I missed the connection between 0.01 and .2 in the format string. Could you please update the comment to reflect the explanation you just gave here?